### PR TITLE
fix player

### PR DESCRIPTION
**Pull request by joaohypolito**

### CODE REVIEW

File: src/main/java/com/squeezecorp/pokermgm/controllers/AuthController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.Optional;
+
+@CrossOrigin(origins = "*")
+@RestController
+@RequestMapping("auth")
+@RequiredArgsConstructor
+public class AuthController {
+    private final PlayerService customerService;
+
+//    @Operation(summary = "Register Player")
```
Comment (andrelake) [inline src/main/java/com/squeezecorp/pokermgm/controllers/AuthController.java:22]: aqui

File: src/main/java/com/squeezecorp/pokermgm/model/PlayerModel.java [hunk 2]
```diff
@@ -12,19 +14,26 @@ public class PlayerModel {
...
 
-    @Column(name = "password", length = 50, nullable = false, unique = true)
+    @Column(name = "password", length = 50, nullable = false, columnDefinition = "VARCHAR(50) COLLATE utf8_bin")
     private String password;
 
-    @ManyToOne
-    @JoinColumn(name = "sessions")
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "sessions", referencedColumnName = "id")
+//    @JsonIgnoreProperties("player_id") // Usar caso definido player dentro de session para evitar serialização
```
Comment (andrelake) [inline src/main/java/com/squeezecorp/pokermgm/model/PlayerModel.java:31]: aqui

File: src/main/java/com/squeezecorp/pokermgm/service/player/PlayerService.java [hunk 3]
```diff
@@ -27,8 +30,9 @@ public PlayerModel createPlayer(PlayerModel player) {
         Optional<PlayerModel> existingPlayer = playerRepository.findByEmail(player.getEmail());
 
         if (existingPlayer.isPresent()) {
-            throw new ResponseStatusException(HttpStatus.NOT_ACCEPTABLE, "Email já cadastrado");
+            throw new ResponseStatusException(HttpStatus.NOT_ACCEPTABLE, "Usuário já cadastrado.");
         }
+//        PlayerModel newPlayer = new PlayerModel(player);
```
Comment (andrelake) [inline src/main/java/com/squeezecorp/pokermgm/service/player/PlayerService.java:35]: aqui

**Review by andrelake — APPROVED** [review]
Deletar comentários

**Review by joaohypolito — COMMENTED** [review]
Commentaries removed.

**Comment by joaohypolito** [comment]
Review and merge if ok.

**Pull request merged**